### PR TITLE
Adding install test for ARM64 on AWS with OVNKube network

### PR DESCRIPTION
**Pull request by aleskandro**
/draft

### CODE REVIEW

**Comment by aleskandro** [comment]
cc @LorbusChris

**Comment by LorbusChris** [comment]
I wonder whether we want this test on the testgrid and triggered by release-controller as well
/cc @deepsm007

File: ci-operator/step-registry/ipi/aws/ovn/ipi-aws-ovn-workflow.metadata.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+		"approvers": [
+			"aravindhp",
+			"ravisantoshgudimetla"
+		],
+		"reviewers": [
+			"aravindhp",
+			"ravisantoshgudimetla"
+		]
+	}
+}
```
Comment (LorbusChris) [inline ci-operator/step-registry/ipi/aws/ovn/ipi-aws-ovn-workflow.metadata.json:13]: nit: missing newline

File: ci-operator/step-registry/ipi/aws/ovn/pre/ipi-aws-ovn-pre-chain.metadata.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+		"approvers": [
+			"aravindhp",
+			"ravisantoshgudimetla"
+		],
+		"reviewers": [
+			"aravindhp",
+			"ravisantoshgudimetla"
+		]
+	}
+}
```
Comment (LorbusChris) [inline ci-operator/step-registry/ipi/aws/ovn/pre/ipi-aws-ovn-pre-chain.metadata.json:13]: nit: missing newline

File: ci-operator/step-registry/ipi/aws/ovn/ipi-aws-ovn-workflow.metadata.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+	"path": "ipi/aws/ovn/ipi-aws-ovn-workflow.yaml",
+	"owners": {
+		"approvers": [
+			"aravindhp",
+			"ravisantoshgudimetla"
+		],
+		"reviewers": [
+			"aravindhp",
```
Comment (LorbusChris) [inline ci-operator/step-registry/ipi/aws/ovn/ipi-aws-ovn-workflow.metadata.json:9]: @aravindhp @ravisantoshgudimetla are you ok with taking excluslve ownership here, or should this list be expanded?

**Review by LorbusChris — CHANGES_REQUESTED** [review]

**Comment by deepsm007** [comment]
> I wonder whether we want this test on the testgrid and triggered by release-controller as well
> /cc @deepsm007

This test provides value and can be added to the testgrid. As for release-controller I leave it upto you.

File: ci-operator/step-registry/ipi/aws/ovn/ipi-aws-ovn-workflow.metadata.json (hunk 3 above)
Comment (aravindhp) [reply ci-operator/step-registry/ipi/aws/ovn/ipi-aws-ovn-workflow.metadata.json:9]: I suggest removing @ravisantoshgudimetla and adding @sebsoto and @JacobTanenbaum.

**Comment by LorbusChris** [comment]
@aleskandro see this PR for an example of adding the test to testgrid: https://github.com/openshift/release/pull/21646

**Review by aravindhp — COMMENTED** [review]
LGTM for the step registry and ownership changes

/cc @JacobTanenbaum

File: ci-operator/step-registry/ipi/aws/ovn/ipi-aws-ovn-workflow.metadata.json (hunk 3 above)
Comment (aleskandro) [reply ci-operator/step-registry/ipi/aws/ovn/ipi-aws-ovn-workflow.metadata.json:10]: Done

File: ci-operator/step-registry/ipi/aws/ovn/pre/ipi-aws-ovn-pre-chain.metadata.json (hunk 2 above)
Comment (aleskandro) [reply ci-operator/step-registry/ipi/aws/ovn/pre/ipi-aws-ovn-pre-chain.metadata.json]: Done

File: ci-operator/step-registry/ipi/aws/ovn/ipi-aws-ovn-workflow.metadata.json (hunk 1 above)
Comment (aleskandro) [reply ci-operator/step-registry/ipi/aws/ovn/ipi-aws-ovn-workflow.metadata.json]: DOne

File: ci-operator/step-registry/ipi/aws/ovn/pre/ipi-aws-ovn-pre-chain.metadata.json (hunk 2 above)
Comment (aleskandro) [reply ci-operator/step-registry/ipi/aws/ovn/pre/ipi-aws-ovn-pre-chain.metadata.json:15]: @LorbusChris adding newline on those metadata JSON files makes the CI failing on ci/prow/step-registry-metadata

New lines at the end of those metadata files are removed by the `make update` jobs

**Review by aravindhp — APPROVED** [review]
/approve

For the step registry and ownership changes

**Comment by aleskandro** [comment]
Hello, failed tests should be a flake. Can I use /retest here?

**Review by stevekuznetsov — APPROVED** [review]

**Comment by aravindhp** [comment]
/hold

@aleskandro please ensure `make jobs` is run to generate updated config

**Comment by aleskandro** [comment]
> 
> @aleskandro please ensure `make jobs` is run to generate updated config

Hi @aravindhp, I needed a rebase of the PR to add the latest commits from origin/master. It should now be ok.

/hold cancel

**Pull request merged**